enterprise/server/auth/auth.go

```diff
@@ -113,7 +115,17 @@ func (c *Claims) IsAdmin() bool {
 	return false
 }
 
-func assembleJWT(ctx context.Context, userID, groupID string, allowedGroups []string) (string, error) {
+func (c *Claims) HasCapability(cap akpb.ApiKey_Capability) bool {
+	return int32(cap)&c.Capabilities > 0
```

Comment:
It seems like we should keep the "int32 storage" decision inside the db + capabilities library, rather than exposing it to this code. WDYT about making Claims.Capabilities a list of capabilities instead?

Reply:
I wasn't sure whether the JSON lib would be able to handle that. Do you know whether that needs to be one of the `json` annotated fields, and if so, how do we exercise that code path?

Reply:
NVM, turns out the JSON lib can deserialize enum lists just fine. Done